runner/providers/provider-github.ts

```diff
@@ -42,14 +45,23 @@ export async function getRepository(project: Project) {
   }
 }
 
-export async function getRepositories(config: Config) {
+export async function getRepositories(
+  config: Pick<
+    Config,
+    'organization' | 'amountOfRepos' | 'repositories' | 'excludeRepos'
+  >
+) {
   const { organization } = await client.request(QUERY_GITHUB_REPOSITORIES, {
     organization: config.organization,
     amountOfRepos: config.amountOfRepos ?? DEFAULT_AMOUNT_OF_REPOS,
   })
-  const orgRepositories: string[] = organization.repositories.nodes.map(({ name }: { name: string }) => `${config.organization}/${name}`)
+  const orgRepositories: string[] = organization.repositories.nodes.map(
+    ({ name }: { name: string }) => `${config.organization}/${name}`
+  )
   const repos = config.repositories ?? orgRepositories ?? []
```

Comment:
Early return `config.repositories` if it is defined (we don't need to make a request then). Also, I don't think `excludeRepos` should matter when using `repositories` option.